src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.tsx

```diff
@@ -150,14 +153,20 @@ const ProteinsListItemInfo = (props: Props) => {
 
   useEffect(() => {
     const abortController = new AbortController();
-    if (summaryStatsLoadingState === LoadingState.LOADING && !uniprotXref) {
-      // if uniprotXref is absent, we cannot fetch relevant data from PDBe; so pretend that we've successfully completed the request
+    if (
+      summaryStatsLoadingState === LoadingState.LOADING &&
+      !proteinStatsXref
+    ) {
+      // if proteinStatsXref is absent, we cannot fetch relevant data from PDBe; so pretend that we've successfully completed the request
       setSummaryStatsLoadingState(LoadingState.SUCCESS);
       return;
     }
 
-    if (summaryStatsLoadingState === LoadingState.LOADING && uniprotXref) {
-      fetchProteinSummaryStats(uniprotXref.accession_id, abortController.signal)
+    if (summaryStatsLoadingState === LoadingState.LOADING && proteinStatsXref) {
```

Comment:
Missed it before. Why do you call it `proteinStatsXref`? Isn't it simply a protein xref?

Reply:
better to call it displaXref as this is the xref that is chosen to display